.github/workflows/fod-sast-scan.yaml

```diff
@@ -0,0 +1,58 @@
+name: Fortify on Demand Static Application Security Testing
+
+# For every new commit made to the default branch, zip the interesting parts of the code,
+# and submit them to our Fortify on Demand instance.
+# A new Static Application Security Testing (SAST) scan will be made.
+
+on:
+  workflow_call:
+    inputs:
+      fod_release_id:
+        required: true
+        type: string
+      files:
+        required: true
+        type: string
+    secrets:
+      FOD_USER:
+        required: true
+      FOD_PAT:
+        required: true
+
+jobs:
+  FoD-SAST-Scan:
+    runs-on: ubuntu-latest
```

Comment:
Is there a way we can pass the runs-on from the workflow calling it? So it can execute in self-hosted nodes for private repos

Reply:
Looks like I [can't use an Array for input default values](https://github.blog/changelog/2021-11-10-github-actions-input-types-for-manual-workflows/). So, I've added `[self-hosted, automated-checks]` as runner in fd3162b.